### PR TITLE
CPM-916: Add migration for target

### PR DESCRIPTION
**Pull request by pierallard**

### CODE REVIEW

**Review by cemorin — APPROVED** [review]

**Review by mmetayer — APPROVED** [review]
GTM after test

File: upgrades/schema/Version_8_0_20230110131126_update_target_column_for_identifier_generators.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+/**
+ * This migration:
+ * - Adds a target_id column
+ * - Sets the attribute_id in this new column from previous target
+ * - Adds a foreign constraint
+ * - Removes target column
+ *
+ * @copyright 2023 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
```
Comment (mmetayer) [inline upgrades/schema/Version_8_0_20230110131126_update_target_column_for_identifier_generators.php:18]:
```suggestion
 * @license   https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
```

**Review by mmetayer — APPROVED** [review]

File: upgrades/test_schema/Version_8_0_20230110131126_update_target_column_for_identifier_generators_Integration.php [hunk 2]
```diff
@@ -27,19 +27,9 @@ protected function setUp(): void
         $this->connection = $this->get('database_connection');
     }
 
-    public function test_it_does_nothing_if_the_column_already_exists(): void
```
Comment (ValentinMumble) [inline upgrades/test_schema/Version_8_0_20230110131126_update_target_column_for_identifier_generators_Integration.php]: :confident:

Comment (pierallard) [reply upgrades/test_schema/Version_8_0_20230110131126_update_target_column_for_identifier_generators_Integration.php]: I don't care

Comment (ghoussard) [reply upgrades/test_schema/Version_8_0_20230110131126_update_target_column_for_identifier_generators_Integration.php]: You'll break master again @pierallard

**Pull request merged**